test/hummingbot/remote_iface/test_mqtt.py

```diff
@@ -71,6 +69,10 @@ def tearDownClass(cls) -> None:
     def setUp(self) -> None:
         super().setUp()
         self.log_records = []
+        self.original_event_loop = asyncio.get_event_loop()
```

Comment:
@cardosofede I've reworked these tests in this PR https://github.com/hummingbot/hummingbot/pull/6388/files

can you revert these changes and apply that PR first to see if these still need applying? There shouldn't be a need to create a new event loop.

Reply:
will do!